src/types/types.ts

```diff
@@ -33,8 +33,8 @@ export interface ImageRendererProps {
   isHover: boolean;
   objectPosition?: React.CSSProperties["objectPosition"];
   borderRadius?: number;
-  tGP?: number; // the greatest product
-  tMS?: number; // the minimum share
+  size_theGreatestProduct?: number;
+  size_theMinimumShare?: number;
```

Comment:
언더바는 지양하는게 좋을 것 같아요. 구분을 짓고 싶다면 object 형태로 하는 것도 좋아보입니다.


```suggestion
  size?: {
    greatestProduct?: number;
    minimumShare?: number;
  };
```

Reply:
이 interface가 쓰이는 컴포넌트 deprecated 되었습니다 이 인터페이스도 deprecated 처리했고 다음 이슈에서 해당 컴포넌트, interface 없애겠습니다